addons/mail/tests/test_mail_activity.py

```diff
@@ -18,3 +19,69 @@ def test_chatter_activity_tour(self):
             "mail_activity_schedule_from_chatter",
             login="admin",
         )
+
+
+class TestActivityScheduleCommon(HttpCase):
```

Comment:
Would also inherit from MailCase or stuff like that, to re-use already existing mocks, tools and variables